__tests__/tests.ts

```diff
@@ -1468,7 +1468,7 @@ describe('htmlbars-inline-precompile', function () {
         import { precompileTemplate } from "@ember/template-compilation";
         import { setComponentTemplate } from "@ember/component";
         import templateOnly from "@ember/component/template-only";
-        export default setComponentTemplate(precompileTemplate('<HelloWorld @color={{"#ff0000"}} />', { scope: () => ({ HelloWorld }), strictMode: true }), templateOnly());
+        export default setComponentTemplate(precompileTemplate('<HelloWorld @color={{"#ff0000"}} />', { scope: () => ({ HelloWorld }), strictMode: true }), templateOnly(undefined, "foo-bar"));
```

Comment:
Should this be PascalCase, since it's what would get shown in the inspector, and how it would be invoked by a consumer?

Reply:
inspector would auto transform it to pascal case

Reply:
it also doesn't matter if it starts with lower case. so `my-template-only-2` would also be shown as `MyTemplateOnly2`